### ISSUE TITLE
The Loss of Training Process is Very Low!!!

### ISSUE THREAD

**Issue by lovekittynine** [ilovepose/DarkPose#6]
Hi, thanks for your great work! I have tried this work to train on my own dataset with ImageNet-pretrained weights, however, during the training process, the loss is very low(nearly  0.00060 initially ). So, is this phenomenon normal？  and will not leading to gradient vanishing?

**Comment by djangogo** [comment]
It's related to your own dataset, you should modify the learning rate.
